runtime/runtime.rs

```diff
@@ -43,6 +46,8 @@ impl Default for BootstrapOptions {
             is_tty: colors::is_tty(),
             agent_version: format!("zinnia_runtime/{}", env!("CARGO_PKG_VERSION")),
             rng_seed: None,
+            // See https://lotus.filecoin.io/lotus/manage/manage-fil/#public-key-address
+            wallet_address: "t1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za".to_owned(),
```

Comment:
```suggestion
            wallet_address: String::from("t1abjxfbp274xpdqcpuaykwkfb43omjotacm2p3za"),
```

The string type takes ownership, I think it's cleaner to construct it directly

Reply:
I hope this compiles xD

Reply:
FWIW, `.to_owned()` was suggested by Rust Analyzer plugin for VS Code.

I am fine either way and will apply your suggestion.